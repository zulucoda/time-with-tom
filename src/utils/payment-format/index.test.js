import paymentFormat from './index';

describe('Utils - Payment Format - Unit Test', () => {
  test('should paid when true', () => {
    const act = paymentFormat(true);
    expect(act).toMatchSnapshot('yes');
  });
  test('should paid when false', () => {
    const act = paymentFormat(true);
    expect(act).toMatchSnapshot('no');
  });

  test('should null when n/a', () => {
    const act = paymentFormat(null);
    expect(act).toMatchSnapshot('n/a');
  });

  test('should undefined when n/a', () => {
    const act = paymentFormat(undefined);
    expect(act).toMatchSnapshot('n/a');
  });
});
