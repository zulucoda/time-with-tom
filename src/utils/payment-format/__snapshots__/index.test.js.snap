// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils - Payment Format - Unit Test should null when n/a: n/a 1`] = `"no"`;

exports[`Utils - Payment Format - Unit Test should paid when false: no 1`] = `"yes"`;

exports[`Utils - Payment Format - Unit Test should paid when true: yes 1`] = `"yes"`;

exports[`Utils - Payment Format - Unit Test should undefined when n/a: n/a 1`] = `"no"`;
